packages/asc-ui/src/components/Tabs/Tabs.tsx

```diff
@@ -38,18 +42,37 @@ function formatPanelId(id: string) {
   return `panel-${id}`
 }
 
-export const Tabs: React.FC<TabsProps> = ({ label, children }) => {
+export const Tabs: React.FC<TabsProps> = ({
+  label,
+  children,
+  initialTab,
+  ...otherProps
+}) => {
   const allTabs = children.map(({ props }) => props.id)
-  const firstTab = allTabs[0] ?? ''
-  const [selectedTab, setSelectedTab] = useState(firstTab)
+  const foundInitialTab = allTabs.find((id) => id === initialTab)
+  const activeTab = foundInitialTab ?? allTabs[0] ?? ''
+
+  if (initialTab && !foundInitialTab) {
+    // eslint-disable-next-line no-console
+    console.warn(
+      `Warning: you passed a wrong initialTab value to Tabs component. Given initialTab ID: ${initialTab}, length of tabs: ${allTabs.length} `,
```

Comment:
I think `Warning: ` is a bit redundant when using `console.warn`. Also logging the length of the tabs seems no longer relevant.